projects/schematics/src/ng-add/actions/add-dependencies.ts

```diff
@@ -0,0 +1,32 @@
+import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { addPackageJsonDependency, NodeDependency, NodeDependencyType } from '@schematics/angular/utility/dependencies';
+
+const dependenciesToAdd = [
+    {
+        name: 'angular-auth-oidc-client',
+        version: '11.3.0',

```

Comment:
Do we have to update this with every version update? Just a note for me

Reply:
We only have to update this if the mechanism for bootstrapping the auth in our lib changes. but to be consistent I would also update it with every release. I plan to improve that in the next version where the current version is being queries from npm directly via http request.